compiler/tools/fpp-depend/test/def_state_machine.fpp

```diff
@@ -0,0 +1,26 @@
+locate state machine StateMachines.Bar at "StateMachines.fpp"
+
+module DefinitionsAndSpecifiers {
+
+  port Port1(
+          a: U32
+          b: F64
+          )
+
+  active component C {
+
+    async input port dataIn: Port1
+
+    state machine Foo
```

Comment:
For stylistic consistency, I would prefer to avoid using Foo and Bar. Can we use letters like S and T? That's what we do in most of the other test cases.

Reply:
I will use S for state machine definitions and s1 and s2 for state machine instances

Reply:
Sounds good!